stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/logic/channel/internal/ChannelStateLogic.kt

```diff
@@ -646,6 +648,7 @@ internal class ChannelStateLogic(
                     ),
                 )
             }
+        processedMessageIds.add(message.id)
```

Comment:
Should we clear this cache somehow? :)

Reply:
Shouldn't be very big cache, but we can use an LRUCache there. Let me check